.changeset/empty-numbers-live.md

```diff
@@ -0,0 +1,101 @@
+---
+'braid-design-system': major
+---
+
+---
+updated:
+  - Stack
+  - Tiles
+---
+
+**Stack, Tiles:** Remove `divider` support
+
+As part of migrating our layout components to leverage flex gap, the `Stack` & `Tiles` no longer iterate over their children, making `dividers` no longer feasible to implement centrally.
+
+While we could have conditionally maintained this behaviour, it would have resulted in inconsistent edge cases when using `dividers` and not, e.g. if a child component rendered nothing or a hidden element, the `divider` would still be rendered, resulting in an inconsistent layout.
```

Comment:
Feel like the `dividers` prop usage is implied here. Have also reworded it a bit. Does it still describe the problem accurately?

```suggestion
While we could have exposed an API to apply this behaviour conditionally, there are edge cases that cannot be handled correctly without consumer-side rendering logic, such as when child components render nothing or a hidden element.
```